backend/src/graphql/models/GdtEntryList.ts

```diff
@@ -0,0 +1,56 @@
+import { GdtEntry } from './GdtEntry'
+import { ObjectType, Field } from 'type-graphql'
+
+@ObjectType()
+export class GdtSumPerEmail {
+  constructor(email: string, summe: number) {
+    this.email = email
+    this.summe = summe
+  }
+
+  @Field(() => String)
+  email: string
+
+  @Field(() => Number)
+  summe: number
+}
+
+@ObjectType()
+export class GdtEntryList {
+  constructor(json: any) {
+    this.state = json.state
+    this.moreEntrysAsShown = json.moreEntrysAsShown
+    this.ownEntries = []
+    if (typeof json.ownEntries !== undefined) {
+      for (const entry in json.ownEntries) {
+        // eslint-disable-next-line no-console
+        console.log(entry)
+        this.ownEntries.push(new GdtEntry(json.ownEntries[entry]))
+      }
+    }
+    this.gdtSumPerEmail = []
+    for (const email in json.gdtSumPerEmail) {
```

Comment:
`this.gdtSumPerEmail = json.gdtSumPerEmail`

Reply:
I must first add more test data to test with this case